### PR TITLE
Updated README and Rake script for LayerConfiguration.json

### PR DESCRIPTION
**Pull request by jpmcglone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +45,33 @@ These commands will clone Atlas Messenger from Github, configure the Atlas submo
...
+
+Run `rake configure:set_app_id["{YOUR_APP_ID}"]` to set the App ID in your Layerfile, or set it manually by editing 'Layerfile' directly.
+
+Replace `{YOUR_APP_ID}` with your appID, obtained from the [Atlas keys](https://developer.layer.com/projects/keys) page.
+
+##### Identity Provider URL
+
+Run `rake configure:set_identity_provider_url["{YOUR_IDENTITY_PROVIDER_URL}"]` to set the Identity Provider URL in your Layerfile, or set it manually by editing 'Layerfile' directly.
 
 Replace `{YOUR_APP_ID}` with your appID, obtained from the [Atlas keys](https://developer.layer.com/projects/keys) page.
```
Comment (daniel-maness) [inline README.md]: Should be {YOUR_IDENTITY_PROVIDER_URL} and reference the identity provider url

File: README.md [hunk 2]
```diff
@@ -45,15 +45,33 @@ These commands will clone Atlas Messenger from Github, configure the Atlas submo
...
-#### Setting the App ID
+### Setting the App ID
 
-Before running Atlas Messenger from source code you must configure the Layer App ID. To do so, switch to the Project Navigator by selecting the **View** menu > **Navigators** > **Show Project Navigator** (or type `⌘1`) and expand the items **Atlas Messenger** and **Code**. Tap on `ATLMAppDelegate.m` to open the application delegate code and locate the following code near the top of the file:
+Before running Atlas Messenger from source code you must configure the Layer App ID and Identity Provider URL. To do so, follow these steps:
 
-Run `rake configure:set["appID", "{YOUR_APP_ID}"]` to set the appID in your Layerfile, or set it manually by editing 'Layerfile' directly.
+##### App ID
+
+Run `rake configure:set_app_id["{YOUR_APP_ID}"]` to set the App ID in your Layerfile, or set it manually by editing 'Layerfile' directly.
```
Comment (feifanzhou) [inline README.md]: Not `Layerfile` … we should also introduce the `LayerConfiguration.json` file. Like a sentence on what it does and the fact that it goes in the project root

File: README.md [hunk 3]
```diff
@@ -45,15 +45,33 @@ These commands will clone Atlas Messenger from Github, configure the Atlas submo
...
 
 Replace `{YOUR_APP_ID}` with your appID, obtained from the [Atlas keys](https://developer.layer.com/projects/keys) page.
 
-Note: The current Atlas Messenger project is designed to only work with a special `Atlas Messenger` project.  To create this special App ID you must go to the [Atlas build](https://developer.layer.com/dashboard/atlas/build)  page. If you use a different App ID you will see a "Request failed: forbidden" error when you launch the app.
+##### Identity Provider URL
+
+Run `rake configure:set_identity_provider_url["{YOUR_IDENTITY_PROVIDER_URL}"]` to set the Identity Provider URL in your Layerfile, or set it manually by editing 'Layerfile' directly.
+
+Replace `{YOUR_IDENTITY_PROVIDER_URL}` with your Identity Provider URL, obtained from the [Atlas keys](https://developer.layer.com/projects/keys) page.
```
Comment (feifanzhou) [inline README.md]: That link won't work — let's just add a link to [this repo](https://github.com/layerhq/instastart-identity-provider) for now.

File: README.md (hunk 2 above)
Comment (jpmcglone) [inline README.md]: Good call.

File: README.md (hunk 3 above)
Comment (jpmcglone) [inline README.md]: Thanks. Will fix and merge before Monday :)

**Pull request merged**